plugin-server/src/config/config.ts

```diff
@@ -199,6 +199,8 @@ export function getDefaultConfig(): PluginsServerConfig {
         CDP_CYCLOTRON_BATCH_DELAY_MS: 50,
         CDP_CYCLOTRON_BATCH_SIZE: 500,
 
+        CDP_GOOGLE_ADWORDS_DEVELOPER_TOKEN: '',
```

Comment:
We already have our token available under the `ADWORDS_DEVELOPER_TOKEN` env variable ([slack](https://posthog.slack.com/archives/C01MM7VT7MG/p1731333664145689))

Reply:
Not for this service though. And for this service we should be prefixing CDP stuff as such.